resources/ec2_ebs_volumes.go

```diff
@@ -0,0 +1,121 @@
+package resources
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func Ec2EbsVolumes() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_ec2_ebs_volumes",
+		Resolver:     fetchEc2EbsVolumes,
+		Multiplex:    client.AccountRegionMultiplex,
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Columns: []schema.Column{
+			{
+				Name:     "account_id",
+				Type:     schema.TypeString,
+				Resolver: client.ResolveAWSAccount,
+			},
+			{
+				Name:     "region",
+				Type:     schema.TypeString,
+				Resolver: client.ResolveAWSRegion,
+			},
+			{
+				Name: "volume_id",
+				Type: schema.TypeString,
+			},
+			{
+				Name: "availability_zone",
+				Type: schema.TypeString,
+			},
+			{
+				Name: "create_time",
+				Type: schema.TypeTimestamp,
+			},
+			{
+				Name: "encrypted",
+				Type: schema.TypeBool,
+			},
+			{
+				Name: "fast_restored",
+				Type: schema.TypeBool,
+			},
+			{
+				Name: "iops",
+				Type: schema.TypeInt,
+			},
+			{
+				Name: "kms_key_id",
+				Type: schema.TypeString,
+			},
+			{
+				Name: "multi_attach_enabled",
+				Type: schema.TypeBool,
+			},
+			{
+				Name: "outpost_arn",
+				Type: schema.TypeString,
+			},
+			{
+				Name: "size",
+				Type: schema.TypeInt,
```

Comment:
```suggestion
				Type: schema.TypeBigInt,
```

Reply:
Type is `int32`, so no need to change it to the `TypeBigInt`